Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
```

Comment:
I cannot really understand why do we need `pipfile` when we already have `requirements.txt`

Reply:
My biggest worry is that we are adding here new, different, "requirements" file. We seem to be able to translate from `pipenv` to `requirements.txt` by running manually `pipenv lock -r`, but does dependabot handle that? Or will we have to do it manually? This might lead to conflicts between those two everytime dependabot is run, until we fix them manually.

Reply:
And we cannot/shouldn't get rid of `requirements.txt` neither. It is used by https://github.com/akamai/cli to install requirements

Reply:
Lock file was added per suggestion from dependbot

> Dependabot can't update vulnerable dependencies without a lockfile
The currently installed version can't be determined.
To resolve the issue add a supported lockfile (Pipfile.lock, pyproject.lock or poetry.lock).

Reply:
Apologies for the late response, In past I have mostly used `pipenv` for majority of my work which does make me biased towards it. Personally I would not keep two different types of dependency files for same project. I have not gone through dependabot's documentation but if it can't fix vulnerabilities without a lock file can't we:

1. Only keep pipfile and generate requirements.txt using either pip-tools or pip freeze wherever needed?
2. Or use pipfile in CLI repo as well? Any particular reason we are sticking with requirements.txt there?
3. I am not aware if there are any ways to generate a lock file by only using requirements.txt but we may create a custom build step which can create the lock file but we will still need to manually propagate those changes into requirements.txt as dependabot won't do it (assuming based on Michał's suggestion)

Reply:
Let's stick with `requirements.txt` only.